File: FragmentTest/app/src/main/java/com/geektech/fragmenttest/MainActivity.java
package com.geektech.fragmenttest;

import androidx.appcompat.app.AppCompatActivity;

import android.app.Activity;
import android.os.Bundle;
import android.view.View;

import com.geektech.fragmenttest.databinding.ActivityMainBinding;

public class MainActivity extends AppCompatActivity {
    private ActivityMainBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());
        getSupportFragmentManager().beginTransaction().add(R.id.container, new MainFragment()).commit();
    }
}

File: RvCotlin3/app/src/main/java/com/geektech/rvcotlin3/MainActivity.kt
package com.geektech.rvcotlin3

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.geektech.rvcotlin3.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    lateinit var binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)
    }
}